app/styles/main.scss

```diff
@@ -994,6 +994,10 @@ select {
       color: $night-mode-error-dark
     }
   }
+
+  .reload {
+    background-color: $night-mode-primary-dark;
```

Comment:
could we use `$night-mode-16dp` instead?

Reply:
I was trying to stick with `primary` but that does look better:

![image](https://user-images.githubusercontent.com/703055/82717928-bb788080-9ce2-11ea-9c47-686c1bd9c09c.png)

![image](https://user-images.githubusercontent.com/703055/82717933-c4695200-9ce2-11ea-8885-04920ca9d6a2.png)